cmip6_downscaling/methods/bcsd.py

```diff
@@ -338,31 +360,27 @@ def return_gcm_predict_rechunked(
         predict_period=predict_period,
         bbox=bbox,
     )
-
+    # opts
+    """
+    Q:'s What are the chunks of gcm_train_subset_full_time_ds? Should this just be full_time instead?
+    1. rechunk try: L353
+    2. time part of subset_dataset, after rechunking. Time slicing from subset_dataset func has to happen post-rechunker.
+    """
```

Comment:
1. use comment instead of string
2. hopefully this is fully addressed now and can be removed